starcluster/node.py

```diff
@@ -668,7 +668,7 @@ def export_fs_to_nodes(self, nodes, export_paths):
         self.stop_exporting_fs_to_nodes(nodes, paths=export_paths)
         log.info("Configuring NFS exports path(s):\n%s" %
                  ' '.join(export_paths))
-        nfs_export_settings = "(async,no_root_squash,no_subtree_check,rw)"
+        #nfs_export_settings = "(async,no_root_squash,no_subtree_check,rw)"
```

Comment:
It's my commit mistake. 
I've actually used 
nfs_export_settings = "(sync,rw)" #no_root_squash,no_subtree_check
this setting anyway not related with #518.

This line should not be commented out.